spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -489,13 +486,9 @@ public SparkContext createSparkContext_1() {
     if (execUri != null) {
       conf.set("spark.executor.uri", execUri);
     }
-    if (System.getenv("SPARK_HOME") != null) {
-      conf.setSparkHome(System.getenv("SPARK_HOME"));
```

Comment:
`SPARK_HOME` will be set by spark-submit script. Otherwise spark-submit can not launch spark app properly.